backend/api/Controllers/RobotController.cs

```diff
@@ -625,7 +633,7 @@ public async Task<ActionResult> ResumeMission([FromRoute] string robotId)
         IsarMission isarMission;
         try
         {
-            isarMission = await _isarService.StartLocalizationMission(robot, localizationPose);
+            isarMission = await _isarService.StartLocalizationMission(robot, scheduleLocalizationMissionQuery.LocalizationPose);
```

Comment:
See comment above, get the LocalizationPose from the deck you got from assetDeckService rather than scheduleLocalizationMissionQuery.

Reply:
I will look at these, currently I am trying troubleshoot a problem:  ---> Microsoft.Data.Sqlite.SqliteException (0x80004005): SQLite Error 19: 'NOT NULL constraint failed: AssetDecks.AssetCode'. For some reason I am initializing the AssetCode as null somewhere and since it is required so it is not nullable.